cmd/ttn-lw-cli/commands/root.go

```diff
@@ -207,6 +209,40 @@ func optionalAuth() error {
 	return nil
 }
 
+func confirmChoice(warnings []string, force bool) bool {
+	if force {
+		return true
+	}
+	if len(warnings) > 0 {
+		for _, warning := range warnings {
+			logger.Info(warning)
```

Comment:
If it's a warning, use `Warn`.

See my previous review: https://github.com/TheThingsNetwork/lorawan-stack/pull/3467#discussion_r526695724